Azure/azure-cli
# [Service Connector] `az webapp/containerapp/spring connection create/update`: Store configurations in App Config

**Related command**
az webapp/containerapp/spring connection create/update <targer-service-type> --appconfig-id <app-config-id>

**Description**
This PR supports writing configurations that Service Connector provides into App Config. A connection from source resource to App Config is automatically built if it doe not already exist to support read/write operations of App Config. 

**Testing Guide**
Create connection: az webapp connection create storage-blob --connection {connection-name} --source-id {source-id} --target-id {target-id} --secret --client-type python --appconfig-id {appconfig-id}
After creation, check the key-value pair stored in Appp Config: az appconfig kv list --name {config_store_name} --key {configuration_name}

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
